README.md

```diff
@@ -59,9 +59,10 @@ written to the `out` object.
 
 *Threat*: A malicious guest can fill up the host disk by generating a high
 amount of data to be written to the serial output.  
-*Mitigation*: There is no mitigation implemented at the serial console emulation
-level. To mitigate this at the VMM level, it is recommended to use as output a
-resource that has a fixed size (e.g. ring buffer or a named pipe).  
+*Mitigation*: A backpressure is triggered by using as output a
```

Comment:
For this particular threat, there is still no mitigation at the serial console emulation, i.e. the guest is still able to fill up the host disk. I think the *Mitigation* section should remain unchanged, and basically we should just remove the *Limitations* one since that one was actually fixed by your PR. Does it make sense? (Also, sorry if I misguided you earlier :( didn't look very carefully then).

Reply:
Thanks for your comment, I've updated the commit and removed the Limitation section!